modules/ROOT/pages/request-reply-scope.adoc

```diff
@@ -41,41 +41,53 @@ Consider a very simple application:
-This is important to note because if the requested flow would have another VM outbound endpoint at the end pointing to a different flow, the VM processor inside the reply scope would still receive the message (or messages) from the triggered flow. +
-You can choose to disable this behavior by setting a MULE_REPLYTO variable:
+To achieve this behavior, Mule implicitly sets a `MULE_REPLYTO` message property to point to the processor in the reply section of the scope. +
+In this case, the VM inbound-endpoint processor in requestFlow's source receives the message with the `MULE_REPLYTO_REQUESTOR` property set to `"reply"`, so Mule sends the resulting message (or messages) back to the requester.
+
+== Using VM Connectors in the Requested Flow
+
+Consider a scenario where you have a Request-Reply scope and the requested flow has a VM outbound endpoint at the end pointing to a different flow.
+In this case, the VM processor inside the reply scope still receives the message (or messages) from the triggered flow.
+
+To disable this behavior, you must set the `MULE_REPLYTO_STOP` variable to `true`:
 
 [source,xml,linenums]
 ----
 <flow name="sampleFlow">
-  <http:listener config-ref="HTTP_Listener_Configuration" path="/"/>
+  <http:listener config-ref="HTTP_Listener_Configuration" path="/start"/>
   <request-reply doc:name="Request-Reply">
-   <vm:outbound-endpoint exchange-pattern="one-way" path="request"/>
-   <vm:inbound-endpoint exchange-pattern="one-way" path="reply" />
+   <vm:outbound-endpoint exchange-pattern="one-way" path="slice"/>
+   <vm:inbound-endpoint exchange-pattern="one-way" path="complete" />
   </request-reply>
 </flow>
 
-<flow name="requestFlow">
-  <vm:inbound-endpoint exchange-pattern="one-way" path="request"/>
-  <set-variable variableName="MULE_REPLYTO_STOP" value="true"/> //<1>
-  <logger message="#[payload]" level="INFO"/>
+<flow name="slicerFlow">
+  <vm:inbound-endpoint exchange-pattern="one-way" path="slice" />
+  <!-- Set Variable MULE_REPLYTO_STOP = true -->
+  <set-variable variableName="MULE_REPLYTO_STOP" value="true" /> //<1>
+  <collection-splitter />
+  <vm:outbound-endpoint exchange-pattern="one-way" path="enhance" />
+</flow>
+
+<flow name="enhancerFlow">
+  <vm:inbound-endpoint exchange-pattern="one-way" path="enhance" />
+  <collection-aggregator failOnTimeout="true" />
+  <vm:outbound-endpoint exchange-pattern="one-way" path="complete" />
 </flow>
 ----
 
-<1> Setting MULE_REPLYTO_STOP to true, tell Mule to not configure the  MULE_REPLYTO_REQUESTOR property in the received message.
+<1> Setting `MULE_REPLYTO_STOP` to `true`, prevents Mule from configuring the `MULE_REPLYTO_REQUESTOR` property in the received message.
 
+== Limitations
 
-[IMPORTANT]
-You can't use transactions in VM and JMS connectors inside a request-reply scope.
+You can't use transactions in VM and JMS connectors inside a request-reply scope. Transactions are not compatible with the way the request-reply scope works.
```

Comment:
Note that this text uses lowercase for "request-reply scope."

After reading through this, I think we want to refer to/name the scope "the Request-Reply scope" but can then refer to "request-reply" by itself (without "the" or "scope"). Does that make sense?